enemy_items/javelin.gd

```diff
@@ -0,0 +1,55 @@
+extends Item
```

Comment:
This file needs comments. As someone not familiar with the code base, I have little to no clue what's going on.